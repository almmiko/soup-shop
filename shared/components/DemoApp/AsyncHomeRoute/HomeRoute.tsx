import React from 'react';
// import Helmet from 'react-helmet';

// import config from '../../../../config';

function HomeRoute() {
  return <div>some</div>
}

export default HomeRoute;
