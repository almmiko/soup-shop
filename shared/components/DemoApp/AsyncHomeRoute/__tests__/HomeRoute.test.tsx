import React from 'react';
import { shallow } from 'enzyme';
import {} from 'jest';

import HomeRoute from '../HomeRoute';

describe('<HomeRoute />', () => {
  test('renders', () => {
    const wrapper = shallow(<HomeRoute />);
    expect(wrapper).toMatchSnapshot();
  });
});
