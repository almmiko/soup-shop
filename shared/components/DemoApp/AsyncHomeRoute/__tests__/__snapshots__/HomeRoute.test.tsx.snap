// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeRoute /> renders 1`] = `
<div>
  some
</div>
`;
